code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -486,8 +638,8 @@
 
 	suicide_messages = list(
 		"is losing branches!",
-		"is pulling themselves apart!",
-		"pulls out a secret stash of herbicide and takes a hearty swig!")
+		"pulls out a secret stash of herbicide and takes a hearty swig!",
```

Comment:
A bad merge ate all the suicide messages and I had to put them back in, I guess I got the order wrong though.